src/interfaces/filing-interfaces/incorporation-interfaces.ts

```diff
@@ -25,6 +25,7 @@ export interface IncorporationApplicationIF {
   incorporationAgreement: {
     agreementType: string
   }
+  shareClasses?: ShareClassIF[] // old schema; only use for loading old filings!
```

Comment:
This simplifies the parsing of old IAs (in a correction). Basically this interface supports both the old and new schemas.